wfe/wfe_test.go

```diff
@@ -1826,6 +1823,19 @@ type mockSADifferentStoredKey struct {
 	core.StorageGetter
 }
 
+func TestLogPayload(t *testing.T) {
```

Comment:
Can you add a comment describing what this test does? Just one sentence about the overall purpose would be great.